roborock/code_mappings.py

```diff
@@ -438,6 +458,17 @@ class RoborockMopIntensityS8MaxVUltra(RoborockMopIntensityCode):
     custom_water_flow = 207
 
 
+class RoborockMopIntensitySaros10R(RoborockMopIntensityCode):
+    off = 200
+    low = 201
+    medium = 202
+    high = 203
```

Comment:
Are these actually called "low" "medium" and "high" for you in the Roborock app? For me it's "mild" "standard" and "Intense".

Reply:
![Screenshot_20250721_215417_Roborock](https://github.com/user-attachments/assets/a4425687-2ee7-4716-a08c-9de10111ad94)
![Screenshot_20250721_215412_Roborock](https://github.com/user-attachments/assets/f00ea0ca-40ad-4506-9c3b-400a81295a97)
![Screenshot_20250721_215405_Roborock](https://github.com/user-attachments/assets/76c59f80-d794-4154-9011-faea29c15230)
![Screenshot_20250721_215359_Roborock](https://github.com/user-attachments/assets/43ff50a1-4ace-4664-be0b-c2d8637760bd)
![Screenshot_20250721_215354_Roborock](https://github.com/user-attachments/assets/c3e41a71-5335-4d3c-8642-02ec1bc07d31)
![Screenshot_20250721_215337_Roborock](https://github.com/user-attachments/assets/ada86b3b-c44f-4a8e-b8ee-4e5acd383c0f)